assets/js/components/product-control/style.scss

```
@@ -0,0 +1,43 @@
+.woocommerce-search-product__item {
+	.woocommerce-search-list__item-name {
+		.description {
+			display: block;
+		}
+	}
+
+	&.is-searching,
+	&.is-skip-level {
+		.woocommerce-search-list__item-prefix:after {
```

We seem to be using the double colon notation everywhere else, I think it would be good to do it here too to keep consistency.
```suggestion
		.woocommerce-search-list__item-prefix::after {
```